.github/workflows/test.yml

```diff
@@ -40,10 +40,8 @@ jobs:
         run: |
           python -m pip install flit
           python -m flit install --symlink
-      - uses: pre-commit/action@v3.0.0
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
https://github.com/pre-commit/action/tree/v3.0.1/?tab=readme-ov-file#using-this-action-in-private-repositories